lib/dotcom_web/components/trip_planner/walking_leg.ex

```diff
@@ -0,0 +1,54 @@
+defmodule DotcomWeb.Components.TripPlanner.WalkingLeg do
+  @moduledoc """
+  A walking leg of a trip.
+  Includes a summary of the walk (time and distance) and a list of steps.
+  """
+
+  use Phoenix.Component
+
+  import MbtaMetro.Components.Accordion, only: [accordion: 1]
+  import MbtaMetro.Components.Icon, only: [icon: 1]
+  import MbtaMetro.Components.List, only: [list: 1]
+
+  alias OpenTripPlannerClient.Schema.Step
+
+  @doc """
+  Renders a walking leg.
+
+  Must be given a `leg` with a:
+    - `distance` (float) in miles
+    - `duration` (integer) in minutes
+    - `mode` with a list of `steps` (list of `Step` structs)
+    - `polyline` (string) to use as the accordion ID
+  """
+
+  attr :leg, :map, required: true
+
+  def walking_leg(assigns) do
+    ~H"""
+    <hr class="my-2 border-t-solid border-slate-200" />
```

Comment:
heh it's different because you used `border-slate-300` in the accordion! anyways, it strikes me as kinda funky that some of the bordering is already baked into the component but this other one we needed to add here. hopefully that'll look better in a followup.

Reply:
Well, this line isn't a part of the accordion. It's a separator from the rest of the steps. So, it shouldn't be part of the accordion.